examples/discrete_turbines/tidal_array.py

```diff
@@ -75,16 +75,25 @@
     farm_options_AR2000.turbine_options.A_support = 2.6 * 14.0  # cross-sectional area of support structure
 farm_options_AR2000.turbine_options.diameter = 20
 farm_options_AR2000.upwind_correction = True  # See https://arxiv.org/abs/1506.03611 for more details
-turbine_density = Function(FunctionSpace(mesh2d, "CG", 1), name='turbine_density_AR2000').assign(0.0)
+turbine_density = Function(FunctionSpace(mesh2d, "CG", 1), name='turbine_density_AR2000')
 farm_options_AR2000.turbine_density = turbine_density
 farm_options_AR2000.turbine_coordinates = [[Constant(x), Constant(y)]
                                            for x in numpy.arange(1000, 1061, 60)
                                            for y in numpy.arange(260, 341, 40)]
+# We can also use estimate the rotor-averaged velocity using a power law, albeit with some significant caveats when
+# turbines are in one another's wake. Note that this is still a depth-averaged example and this is just an estimation!
```

Comment:
"Note that this is an estimation to estimate the influence of position along the water column in a depth-averaged" -